.eslintignore

```diff
@@ -9,3 +7,5 @@ data/
 bin/prerender.js
 bin/prerender.js.map
 aboutlibrary/content/
+node_modules/
+**/*.json
```

Comment:
Do we need to add these? The latter we just don't include in `--ext` and `node_modules` should automatically be ignored. Looks like we can clean up a bit more for a minimal list right now:

```
bin/prerender.js                                                                                       
data/                                                                                                  
logs/                                                                                                  
prerendered/                                                                                           
vendor/
```
(And prefixing this minimal list with `browser/components/newtab/` seems to be good for `./mach lint` too.)